src/functions/indGraph.jl

```diff
@@ -0,0 +1,79 @@
+export IndGraph
+
+"""
+**Indicator of an operator graph**
```

Comment:
Okay, sure, I was quite ignorant about how doc works, so I just copied the description from your `IndAffine` PR and thought that it might be enought :)